addons/calendar/models/calendar_recurrence.py

```diff
@@ -216,6 +216,10 @@ def _apply_recurrence(self, specific_values_creation=None, no_send_edit=False):
             events_to_keep, ranges = recurrence._reconcile_events(ranges)
             keep |= events_to_keep
             [base_values] = event.copy_data()
+            base_values['attendee_ids'] = [
+                (0, 0, dict(state='needsAction', partner_id=a['partner_id'].id))
+                for a in event['attendee_ids']
+            ]
```

Comment:
Hello,
Thanks for your fix but I am not found of it. You change the behavior when google_calendar is not installed. 
As @remybar  commented, what happens now when you change a single event and make it recurrent?
Right now, the event is automatically accepted for you and others are "Need action". Which is fine. Maybe they are not available every time. But the user who created the event from the interface should automatically accept it. In V14.0 there is no way to automatically accept all the events of the recurrence. Degrading this flow will be super annoying to user not syncing with google.

You should check if you could override the base_values['attendee_ids'] only if you comes from google_calendar. Maybe with a well defined context key. In V15.0, I introduced the generic value creation to override these kind of things but you can't change the signature in stable.

Reply:
Thanks for your feedback.

I got your point. In my first fix, I used the base event's status for the rest, but @remybar suggested this. However, as the second part of the fix prevents sending synced recurrent events, we can ignore the first part of the fix, and I'll remove it.